src/Channels/Channel.cs

```diff
@@ -52,17 +52,21 @@ public Channel(IBufferPool pool)
         /// <summary>
         /// Gets a task that completes when the channel is completed and has no more data to be read.
         /// </summary>
-        public Task WriterCompleted => _readingTcs.Task;
+        public Task Writing => _readingTcs.Task;
+
+        public bool IsWriterFinished => _readingTcs.Task.IsCompleted;
```

Comment:
The interface inverts the meaning :-/

However do the implemeters also invert the meanings; so they are actually the same?